erpnext/manufacturing/doctype/bom/bom.py

```diff
@@ -578,6 +578,8 @@ def validate_operations(self):
 			for d in self.operations:
 				if not d.description:
 					d.description = frappe.db.get_value('Operation', d.operation, 'description')
+				if not d.is_batch_operation:
+					d.batch_size = 1
```

Comment:
Why batch size is hardcoded? let's keep it this as default in doctype

Reply:
I've made the other required changes. Also for this I've set the default value to 1 in the doctype. I've just added this check to ensure if anything less than 1 is entered it defaults back to 1.